src/main/java/se/lfv/reqstool/plugins/maven/RequirementsToolMojo.java

```diff
@@ -83,6 +83,9 @@ public class RequirementsToolMojo extends AbstractMojo {
 	@Parameter(property = "surefireReportsDir", defaultValue = "${project.build.directory}/surefire-reports")
 	private File surefireReportsDir;
 
+	@Parameter(property = "projectReqstoolDir", defaultValue = "${project.build.finalName}-reqstool")
+	private String projectReqstoolDir;
+
 	@Parameter(property = "zipArtifactFilename", defaultValue = "${project.build.finalName}-reqstool.zip")
 	private String zipArtifactFilename;
 
```

Comment:
We have an issue here since these can be set differently.

I think it's better that the zipArtifactFilename is not configurable.

And the zipArtifactTopLevelDir (i.e. projectReqstoolDir) should not be configurable either.

Use this.project.build.finalName in the code when needed